pkg/virt-controller/services/template_test.go

```diff
@@ -2905,6 +2905,7 @@ var _ = Describe("Template", func() {
 				vmi.Spec.Networks = append(vmi.Spec.Networks, *v1.DefaultPodNetwork())
 				vmi.Spec.Domain.Devices.Interfaces = append(vmi.Spec.Domain.Devices.Interfaces, *v1.DefaultSlirpNetworkInterface())
 
+				config, kvInformer, svc = configFactory(defaultArch)
```

Comment:
Isnt this commit redundant? those tests get removed in follow up commit.

Reply:
The intermediate work during the refactoring failed due to this, so they got fixed.
It is not possible to refactor properly without fixing stuff, and we do not always know what the end result will be.

When the author is asked to go back and change something, the tests may fail without such fixes, making the work harder.